main/getting-started/start-a-project.md

```diff
@@ -22,24 +22,23 @@ Also, for what needs to be done after you modify a project's code, see
   </th>
   <tr>
     <td><center>1</center></td>
-    <td>Go to or open a shell and <code>cd ~/agoric-sdk</code></td>
-    <td>Go to the directory where you copied the latest Agoric SDK
-	    from the Agoric GitHub repository in Step 4 of <a href="https://agoric.com/documentation/getting-started/before-using-agoric/">Before Using Agoric</a>.</td>
+    <td>Go to or open a shell and <code>cd &lt;directory-where-you-want-to-install-Dapp-code&gt;</code></td>
+    <td>When you initialize your project/Dapp in the next step, its initial code is copied into your current directory (permissions allowing).</td>
```

Comment:
This seems random. there's no "initialize" operation. The `init` operation does copy code into the current directory. It creates a new named directory and copies it into there.